src/index.html

```diff
@@ -1,13 +1,493 @@
+                class="about-creators__img"
+              >
+            </picture>
+          </div>
+        </div>
+      </section>
+
+      <!-- <section class="component page__component">
+          <h2 class="title component__title">Ayurvedic Tooth Powder</h2>
+
+          <div class="component__details">
+            <ul class="component__ingridients">
+              <li class="paragraph component__ingridient">Kaolin</li>
+              <li class="paragraph component__ingridient">Sivak <span>(Persian Salvador)</span></li>
+              <li class="paragraph component__ingridient">Calcium Carbonate</li>
+              <li class="paragraph component__ingridient">Licorice Root</li>
+              <li class="paragraph component__ingridient">Turmeric</li>
+            </ul>
+
+            <img src="images/jar_big.png" alt="Powder" class="component__img">
+          </div>
+
+          <a href="#" class="link component__link">Learn more</a>
+      </section> -->
+
+      <section class="benefits page__benefits">
+        <div class="container benefits__content">
+          <div class="benefits__item">
+            <div class="benefits__img-wrapper">
+              <img
+                src="images/element1.png"
+                alt="Snail Icon"
+                class="benefits__img"
+              >
+            </div>
+            <p class="text text--sm benefits__text">
+              No synthetics and microplastics
+            </p>
+          </div>
+
+          <div class="benefits__item">
+            <div class="benefits__img-wrapper">
+              <img
+                src="images/element2.png"
+                alt="Recycled Icon"
+                class="benefits__img"
+              >
+            </div>
+            <p class="text text--sm benefits__text">
+              Product packaging can be recycled or reused
+            </p>
+          </div>
+
+          <div class="benefits__item">
+            <div class="benefits__img-wrapper">
+              <img
+                src="./images/element3.png"
+                alt="Leaves Icon"
+                class="benefits__img"
+              >
+            </div>
+            <p class="text text--sm benefits__text">
+              Only natural ingredients
+            </p>
+          </div>
+        </div>
+      </section>
+
+      <section class="contact-us page__contact-us" id="contact">
+        <div class="container">
+          <div class="contact-us__content-wrapper">
+            <article class="contact-us__content">
+              <h2 class="title contact-us__title">Contact us</h2>
+              <p class="text contact-us__text">
+                We will help you choose a product that suits you and answer your
+                other questions
+              </p>
+              <form
+                action="#"
+                method="GET"
+                class="contact-us__form form"
+              >
+                <label class="form__item" for="name">
```

Comment:
Why do you use label for input? You have only input without text.

Reply:
Thank you very much for your comments)